### PR TITLE
AO3-4268 Implement full screen width dropdowns for small screen sizes

### PR DESCRIPTION
**Pull request by weeklies**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [ ] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-4268

## Purpose

Make dropdown menus fit small screens.

## Testing Instructions
See Jira ticket.
## Credit

weeklies (she/her)

### CODE REVIEW

**Comment by bingeling** [comment]
Hi weeklies,

thank you for your PR! We have discussed internally what behaviour we want for this. We would prefer a solution that only applies to the narrow media query, not universally. And that in this case the positioning of the dropdown panel is turned off, instead the panel takes up 100% of the screen width, from left side of the screen to the right side. Can you make these adjustments?

**Comment by weeklies** [comment]
> Hi weeklies,
> 
> thank you for your PR! We have discussed internally what behaviour we want for this. We would prefer a solution that only applies to the narrow media query, not universally. And that in this case the positioning of the dropdown panel is turned off, instead the panel takes up 100% of the screen width, from left side of the screen to the right side. Can you make these adjustments?

Sure, will make the changes.

File: public/stylesheets/site/2.0/26-media-narrow.css [hunk 1]
```diff
@@ -74,6 +74,16 @@ body .narrow-shown {
   margin-left: 45px;
 }
 
+#header .dropdown, #greeting .user {
+  position: static;
+}
+
+#header .menu {
+  width: 100vw;
```
[the remark this answers is not in the document]
Comment (weeklies) [reply public/stylesheets/site/2.0/26-media-narrow.css:82]: Not really, thanks for letting me know.

**Comment by bingeling** [comment]
This looks good, thank you!

**Review by bingeling — APPROVED** [review]

**Pull request merged**